### PR TITLE
Porting BottomSheet accessibility from v2 to v4

### PR DESCRIPTION
**Pull request by Mahmoud-SK**
## Motivation

I'm picking up this [task](https://github.com/gorhom/react-native-bottom-sheet/pull/693) of porting accessibility features from V2 to V4 as per [#548](https://github.com/gorhom/react-native-bottom-sheet/issues/548)

### CODE REVIEW

**Comment by jenskuhrjorgensen** [comment]
Awesome work <3 

This missing functionality is what is keeping us on v2 of this library!

**Comment by BethThomas141** [comment]
Would love to see this merged in 👍

**Comment by BethThomas141** [comment]
Commenting to keep this open - would be a real help if this gets merged in.

**Comment by BethThomas141** [comment]
@gorhom any chance of this going in soon? 🙏

**Comment by Mahmoud-SK** [comment]
We would very much also like to get this merged in. @gorhom

**Comment by manipatel30** [comment]
@gorhom Is this change available in 4.4.6 ?
Ohh sorry PR is still OPEN

**Comment by jenskuhrjorgensen** [comment]
Please keep this open - we are still longing for it!

**Comment by jenskuhrjorgensen** [comment]
Hi @gorhom 

I'm sure you are super busy maintaining this awesome library, but can we at least get a comment on why this PR is not getting any attention? To me it is a fairly small PR that only adds some accessibility capabilities and doesn't touch any core functionality that risks breaking the lib. 

Please just give us a small sign of life from you and let us know if there is anything we can do in order to get this merged <3

File: src/components/bottomSheet/types.d.ts [hunk 1]
```diff
@@ -295,6 +299,21 @@ export interface BottomSheetProps
...
+   * Enables announcement of the snap position change on screen readers.
+   * @type boolean
+   * @default true
+   */
+  enableAccessibilityChangeAnnouncement?: boolean;
+  /**
+   * Callback on snap position change that announces the returned string on screen readers.
+   * @type string | ((positionInScreen: string) => string)
+   */
+  accessibilityPositionChangeAnnouncement?:
```
Comment (gorhom) [inline src/components/bottomSheet/types.d.ts]: these two props could be handled with `onChange` by the user outside of the library, i don't think we need to handle it internally

**Comment by Mahmoud-SK** [comment]
Hi @gorhom I see you have assigned yourself to this PR some time ago now, do you have any ETA on when we can get this reviewed and approved? 

We are still very much waiting on this. ❤️

**Comment by BethThomas141** [comment]
@gorhom Please can you take a quick look at this and merge it in? My app cares about users with accessibility requirements so we really need this

**Comment by C-Dykes** [comment]
Any news on this? its still an issue

**Comment by jenskuhrjorgensen** [comment]
Hi again @gorhom 

We are currently running on an old forked version of your **_awesome_** lib, but now we have to bump in order to get to use reanimated v3. However, since v4 of this lib still doesn't cater for accessibility users, we will unfortunately have to create a new fork of `react-native-bottom-sheet`, since accessibility support is being enforced in more and more countries. For instance, the European Accessibility Act (EEA) has already been implemented in several European countries including some of the countries that we operate in:

> The [European Accessibility Act](https://eur-lex.europa.eu/legal-content/EN/LSU/?uri=CELEX%3A32019L0882) applies to most private companies. However, it is not yet fully enacted: The legislation will be enforced by July 28, 2025, and member states have until June 28, 2022 to adopt the EAA into their national laws.
Source: https://www.boia.org/blog/european-union-accessibility-laws-an-overview

I believe this PR is fairly simple as it doesn't touch any core or complex functionality but instead just adds some additional accessibility props but if there is something that you would like to have done in a different way, please at least provide us with a review and some feedback on whatever is holding you back from reviewing/approving/merging this PR. If you have been inactive on this PR because you think "accessibility" is just dull and doesn't matter (I don't blame you if you do - I did so too) I think the EEA and below quote shows that accessibility is actually quite important and is more and more becoming a standard requirement/expected feature both by end users as well as developers:

> The percentage of Americans with disabilities is a whopping 26%, which equals roughly one in four adults with disabilities on a population level. To be able to reach as many people as possible, you need to make your website accessible to people with disabilities.
Source: https://monsido.com/blog/accessibility-statistics

Looking forward to hear an answer from you that can hopefully stop our process of forking yet another version of `react-native-bottom-sheet`.

Best regards
Jens

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 2]
```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
...
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
+        {...rest}
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: @Mahmoud-SK why is the spread of the `rest` props needed?

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 3]
```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
...
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: @Mahmoud-SK can we maybe do this in order to not break the existing behaviour?
```
accessibilityHint ={_providedAccessiblityHint  ? _providedAccessibilityHint  : `Tap to ${
          typeof pressBehavior === 'string' ? pressBehavior : 'move'
        } the Bottom Sheet`}
```

File: src/components/bottomSheet/BottomSheet.tsx [hunk 4]
```diff
@@ -323,7 +341,7 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
       const extendedPositionWithKeyboard = Math.max(
         0,
         animatedContainerHeight.value -
-          (animatedSheetHeight.value + keyboardHeightInContainer)
+        (animatedSheetHeight.value + keyboardHeightInContainer)
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/BottomSheet.tsx]: I think this was correctly indented before - can you revert this change?

File: src/components/bottomSheet/BottomSheet.tsx [hunk 5]
```diff
@@ -473,11 +491,11 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
...
-            animatedPosition.value,
-            adjustedSnapPoints,
-            adjustedSnapPointsIndexes,
-            Extrapolate.CLAMP
-          )
+          animatedPosition.value,
+          adjustedSnapPoints,
+          adjustedSnapPointsIndexes,
+          Extrapolate.CLAMP
+        )
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/BottomSheet.tsx]: I think this was correctly indented before - can you revert this change?

File: src/components/bottomSheet/BottomSheet.tsx [hunk 6]
```diff
@@ -1393,9 +1444,9 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
          */
         animatedKeyboardHeightInContainer.value = $modal
           ? Math.abs(
-              _keyboardHeight -
-                Math.abs(bottomInset - animatedContainerOffset.value.bottom)
-            )
+            _keyboardHeight -
+            Math.abs(bottomInset - animatedContainerOffset.value.bottom)
+          )
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/BottomSheet.tsx]: Wasn't this also correctly indented before?

File: src/components/bottomSheet/BottomSheet.tsx [hunk 7]
```diff
@@ -1635,6 +1686,10 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
                 <Animated.View
                   pointerEvents="box-none"
                   style={contentMaskContainerStyle}
+                  accessible={_providedAccessible ?? undefined}
+                  accessibilityRole={_providedAccessibilityRole ?? undefined}
+                  accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+                  {...rest}
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/BottomSheet.tsx]: Why is the spread of the `rest` props needed?

File: src/components/bottomSheet/BottomSheet.tsx [hunk 8]
```diff
@@ -159,6 +165,18 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
...
+      accessible: _providedAccessible = DEFAULT_ACCESSIBLE,
+      accessibilityLabel:
+      _providedAccessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+      accessibilityRole:
+      _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+      enableAccessibilityChangeAnnouncement:
+      _providedEnableAccessibilityChangeAnnouncement = DEFAULT_ENABLE_ACCESSIBILITY_CHANGE_ANNOUNCEMENT,
+      accessibilityPositionChangeAnnouncement:
+      _providedAccessibilityPositionChangeAnnouncement = DEFAULT_ACCESSIBILITY_POSITION_CHANGE_ANNOUNCEMENT,
+      ...rest
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/BottomSheet.tsx]: Why is the spread of the `rest` props needed?

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 9]
```diff
@@ -33,6 +37,11 @@ const BottomSheetBackdropComponent = ({
   onPress,
   style,
   children,
+  accessible: _providedAccessible = DEFAULT_ACCESSIBLE,
+  accessibilityRole: _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+  accessibilityLabel: _providedAccessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+  accessibilityHint: _providedAccessiblityHint = DEFAULT_ACCESSIBILITY_HINT,
+  ...rest
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: Why is the spread of the `rest` props needed?

File: src/components/bottomSheetBackdrop/constants.ts [hunk 10]
```diff
@@ -4,10 +4,19 @@ const DEFAULT_DISAPPEARS_ON_INDEX = 0;
 const DEFAULT_ENABLE_TOUCH_THROUGH = false;
 const DEFAULT_PRESS_BEHAVIOR = 'close' as const;
 
+const DEFAULT_ACCESSIBLE = true;
+const DEFAULT_ACCESSIBILITY_ROLE = 'button';
+const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom Sheet backdrop';
+const DEFAULT_ACCESSIBILITY_HINT = 'Tap to close the Bottom Sheet';
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/constants.ts]:
```suggestion
const DEFAULT_ACCESSIBILITY_HINT = 'Tap to close the bottom sheet';
```

File: src/components/bottomSheetBackdrop/constants.ts [hunk 11]
```diff
@@ -4,10 +4,19 @@ const DEFAULT_DISAPPEARS_ON_INDEX = 0;
 const DEFAULT_ENABLE_TOUCH_THROUGH = false;
 const DEFAULT_PRESS_BEHAVIOR = 'close' as const;
 
+const DEFAULT_ACCESSIBLE = true;
+const DEFAULT_ACCESSIBILITY_ROLE = 'button';
+const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom Sheet backdrop';
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/constants.ts]:
```suggestion
const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom sheet backdrop';
```

File: src/components/bottomSheet/constants.ts [hunk 12]
```diff
@@ -33,6 +33,15 @@ const INITIAL_CONTAINER_OFFSET = {
 const INITIAL_HANDLE_HEIGHT = -999;
 const INITIAL_POSITION = SCREEN_HEIGHT;
 
+// accessibility
+const DEFAULT_ACCESSIBLE = true;
+const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom Sheet';
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheet/constants.ts:38]:
```suggestion
const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom sheet';
```

File: src/components/bottomSheetHandle/BottomSheetHandle.tsx [hunk 13]
```diff
@@ -2,11 +2,22 @@ import React, { memo, useMemo } from 'react';
...
 
 const BottomSheetHandleComponent = ({
   style,
   indicatorStyle: _indicatorStyle,
   children,
+  accessible = DEFAULT_ACCESSIBLE,
+  accessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+  accessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+  accessibilityHint = DEFAULT_ACCESSIBILITY_HINT,
+  ...rest
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetHandle/BottomSheetHandle.tsx]: Why is the spread of the `rest` props needed?

File: src/components/bottomSheetHandle/constants.ts [hunk 14]
```diff
@@ -0,0 +1,12 @@
+const DEFAULT_ACCESSIBLE = true;
+const DEFAULT_ACCESSIBILITY_ROLE = 'adjustable';
+const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom Sheet handle';
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetHandle/constants.ts]:
```suggestion
const DEFAULT_ACCESSIBILITY_LABEL = 'Bottom sheet handle';
```

File: src/components/bottomSheetHandle/constants.ts [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+const DEFAULT_ACCESSIBILITY_HINT =
+  'Drag up or down to extend or minimize the Bottom Sheet';
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetHandle/constants.ts]:
```suggestion
  'Drag up or down to extend or minimize the bottom sheet';
```

**Review by jenskuhrjorgensen — COMMENTED** [review]
Hi @Mahmoud-SK 

I've added some review comments to your PR in order to maybe make the change a little smaller in an attempt to maybe get this merged/prioritized. 

Let me know what you think.

Best regards
Jens

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 16]
```diff
@@ -33,6 +37,11 @@ const BottomSheetBackdropComponent = ({
   onPress,
   style,
   children,
+  accessible: _providedAccessible = DEFAULT_ACCESSIBLE,
+  accessibilityRole: _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+  accessibilityLabel: _providedAccessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+  accessibilityHint: _providedAccessiblityHint = DEFAULT_ACCESSIBILITY_HINT,
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]:
```suggestion
  accessibilityHint: _providedAccessibilityHint = DEFAULT_ACCESSIBILITY_HINT,
```

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 17]
```diff
@@ -119,12 +128,11 @@ const BottomSheetBackdropComponent = ({
...
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
+        accessibilityRole={_providedAccessibilityRole ?? undefined}
+        accessibilityLabel={_providedAccessibilityLabel ?? undefined}
+        accessibilityHint={_providedAccessiblityHint ?? undefined}
+        {...rest}
       >
         {children}
       </Animated.View>
```
Comment (jenskuhrjorgensen) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx:141]: I think you also need to add the accessibility props to the `<Animated.View>` below:
```
    <Animated.View
      style={containerStyle}
      pointerEvents={pointerEvents}
      accessible={_providedAccessible ?? undefined}
      accessibilityRole={_providedAccessibilityRole ?? undefined}
      accessibilityLabel={_providedAccessibilityLabel ?? undefined}
      accessibilityHint={_providedAccessibilityHint ?? undefined}
      {...rest}
    >
      {children}
    </Animated.View>
```
Maybe add this to a local const and use that because it is the exact same component as is used inside `<TapGestureHandler/>` when `pressBehavior !== 'none'`.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 2 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: It's needed to pass the rest of the properties to the Animated.View - in this case it's only animatedPosition-prop.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 3 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: Done. :)

File: src/components/bottomSheet/BottomSheet.tsx (hunk 4 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheet/BottomSheet.tsx]: It looks correctly indented locally, so I'm not completely sure why it shows it like this.

File: src/components/bottomSheet/BottomSheet.tsx (hunk 5 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheet/BottomSheet.tsx]: Same as the previous comment, locally it's indented.

File: src/components/bottomSheet/BottomSheet.tsx (hunk 6 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheet/BottomSheet.tsx]: Same as the previous comment, locally it's indented.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 9 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: It currently contains the animatedPosition-prop. I could just write it out instead of using the spread operator.

File: src/components/bottomSheetBackdrop/constants.ts (hunk 10 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/constants.ts]: Done.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 17 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx:141]: Good idea, done.

**Comment by liviu-padurariu** [comment]
Hi all, is there anything stopping for this branch to be merged?

File: src/components/bottomSheet/BottomSheet.tsx [hunk 18]
```diff
@@ -602,8 +619,42 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
         if (_providedOnChange) {
           _providedOnChange(index);
         }
+
+        AccessibilityInfo.isScreenReaderEnabled().then(isEnabled => {
```
Comment (gorhom) [inline src/components/bottomSheet/BottomSheet.tsx]: lets keep this logic out of the library and let user customise it as they pleased with `onChange` prop.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 19]
```diff
@@ -33,6 +37,11 @@ const BottomSheetBackdropComponent = ({
   onPress,
   style,
   children,
+  accessible: _providedAccessible = DEFAULT_ACCESSIBLE,
+  accessibilityRole: _providedAccessibilityRole = DEFAULT_ACCESSIBILITY_ROLE,
+  accessibilityLabel: _providedAccessibilityLabel = DEFAULT_ACCESSIBILITY_LABEL,
+  accessibilityHint: _providedAccessibilityHint = DEFAULT_ACCESSIBILITY_HINT,
+  ...rest
```
Comment (gorhom) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: please remove `...rest`

File: src/components/bottomSheetHandle/BottomSheetHandle.tsx (hunk 13 above)
Comment (gorhom) [inline src/components/bottomSheetHandle/BottomSheetHandle.tsx]: please remove `...rest`

**Review by gorhom — CHANGES_REQUESTED** [review]
thanks @Mahmoud-SK, for submitting this pr, i have left some comments. and most importantly i think we should not introduce the change announcement, since i believe it is on the user to set it up not the library.

File: src/components/bottomSheet/types.d.ts (hunk 1 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheet/types.d.ts]: I've went ahead and removed these two props.

File: src/components/bottomSheet/BottomSheet.tsx (hunk 18 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheet/BottomSheet.tsx]: I've removed the "AccessibilityInfo.isScreenReaderEnabled"-logic out of the library.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 19 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx]: Removed the ...rest

File: src/components/bottomSheetHandle/BottomSheetHandle.tsx (hunk 13 above)
Comment (Mahmoud-SK) [reply src/components/bottomSheetHandle/BottomSheetHandle.tsx]: Removed the ...rest

**Comment by Mahmoud-SK** [comment]
Hi @gorhom I've went ahead and done the changes that you requested. 😄 Could you please have a look again? 

Thanks in advance!

**Comment by gorhom** [comment]
rebased the pr branch.. going to test the changes on the backdrop then i'll merge it

**Review by gorhom — APPROVED** [review]

**Comment by gorhom** [comment]
thanks @Mahmoud-SK for submitting this PR, i appreciate your amazing work 🙌

this should be release on the next few days

**Comment by jenskuhrjorgensen** [comment]
Awesome news, @gorhom!

The accessibility features were once added to V2 but were not added to newer versions of the lib. Will the changes in this PR be added to V5 or should we create a separate PR for that towards the V5 branch? Just to avoid having to request this feature again for the next major version.

**Comment by gorhom** [comment]
> Awesome news, @gorhom!
> 
> 
> 
> The accessibility features were once added to V2 but were not added to newer versions of the lib. Will the changes in this PR be added to V5 or should we create a separate PR for that towards the V5 branch? Just to avoid having to request this feature again for the next major version. 

I have already cherrypicked this PR into v5 latest release 👏

**Comment by TrustDec** [comment]
> > Awesome news, @gorhom!
> > The accessibility features were once added to V2 but were not added to newer versions of the lib. Will the changes in this PR be added to V5 or should we create a separate PR for that towards the V5 branch? Just to avoid having to request this feature again for the next major version.
> 
> I have already cherrypicked this PR into v5 latest release 👏

Hi, can you take a look at this question? https://github.com/gorhom/react-native-bottom-sheet/issues/1560#issue-1926089538

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx [hunk 20]
```diff
@@ -112,27 +120,35 @@ const BottomSheetBackdropComponent = ({
...
       <Animated.View
         style={containerStyle}
         pointerEvents={pointerEvents}
-        accessible={true}
-        accessibilityRole="button"
-        accessibilityLabel="Bottom Sheet backdrop"
-        accessibilityHint={`Tap to ${
-          typeof pressBehavior === 'string' ? pressBehavior : 'move'
-        } the Bottom Sheet`}
+        accessible={_providedAccessible ?? undefined}
```
Comment (aweary) [inline src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx:129]: Why would you mark the root `View` as `accessible` for the bottom sheet? That makes it so you cannot focus any of the descendant views (which is all of the sheet content) with VoiceOver on iOS.

**Review by aweary — COMMENTED** [review]
@gorhom we recently updated our fork of this library and this commit broke VoiceOver for all of our sheets. I don't think this change makes sense for any sheet that would have interactive content in it (which I imagine is most?).

I would recommend reverting, this doesn't improve accessibility at all.

File: src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx (hunk 20 above)
Comment (haileyok) [reply src/components/bottomSheetBackdrop/BottomSheetBackdrop.tsx:129]: Yep, just tested this and this is the problem I'm seeing as well. @flo-sch I see you just opened an accessibility PR, did you want to take a look at this as well there?

**Comment by haileyok** [comment]
@gorhom We also had to revert this for the same reasons aweary mentioned. I think it would probably be a good idea revert as well, especially if this has made it's way into v5.
